pkg/cloud/services/ec2/securitygroups.go

```diff
@@ -18,6 +18,7 @@ package ec2
 
 import (
 	"fmt"
+
 	"sigs.k8s.io/cluster-api/util/conditions"
 
 	errlist "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
similar here for imports, these should be consolidated (and also consolidated with the grouping below)